routes/api.php

```diff
@@ -14,6 +16,13 @@
 |
 */
 
-Route::middleware(['auth:sanctum'])->get('/user', function (Request $request) {
-    return $request->user();
-});
+// Auth routes
+Route::post('/login', [AuthController::class, 'login'])->name('login');
+Route::post('password/email', ForgotPasswordController::class)->name('password.send-email');
+Route::post('password/reset', ResetPasswordController::class)->name('password.reset');
+
+Route::middleware(['auth:sanctum'])
+    ->group(function () {
+        // Auth routes
+        Route::post('/logout', [AuthController::class, 'logout'])->name('logout');
```

Comment:
Why don't you add the logout route in the same block as the other Auth routes (outside this middleware). You can add the middleware isolated just for the logout route just to keep it grouped:

` Route::post('/logout', [AuthController::class, 'logout'])->name('logout')->middleware('['auth:sanctum']);`

We then leave this Route middleware group for user-space routes.